contracts/id.tonomy/include/id.tonomy/id.tonomy.hpp

```diff
@@ -82,21 +83,41 @@ namespace idtonomy
           public_key password_key,
           checksum256 password_salt);
 
+      /**
+       * Create a new account for an app and registers it's details
+       *
+       * @details Creates a new account for an app and registers it's details.
+       *
+       * @param name - name of the app
+       * @param description - description of the app
+       * @param username_prefix - username prefix
+       * @param logo_url - url to the logo of the app
+       * @param domain - domain associated with the app
+       * @param password_key - public key generated from the account's password
+       */
+      [[eosio::action]] void newapp(
+         string name,
+         string description,
+         checksum256 username_hash,
```

Comment:
There's a username hash here, but it's not in the details? but it is in line 146.
 Or is this the Username prefix?
If this is the way it should be, then this file looks good to me!